PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
+def perilEndorsement():
+    print "Getting authorization token."
+
+    #URL = "https://api.develop.socotra.com/account/authenticate"
+    URL = "http://127.0.0.1:8080/account/authenticate"
+    #URL = "https://api.develop.socotra.com/account/v1/authenticateInternal
+    payload = {
+                "username": "alice.lee",
+                "password": "socotra",
+                "hostName": "vishnu-vishnu1.co.develop.socotra.com"
+                }
+
+    r = requests.post(URL, json=payload)
+    data = json.loads(r.text)
+    authorizationToken = data["authorizationToken"]
+    print("Retrieved the authorization token.")
+
+    policyResponseURL = "https://api.develop.socotra.com/policy/100005202/"
```

Comment:
The URL, payload, policy_id, etc. should probably be parameterized and not hard-coded. You could load the configuration from place in the bottom from the file, a separate config file or, better still use Argparse (https://docs.python.org/3/library/argparse.html).